src/pages/billing/Project.js

```diff
@@ -308,7 +319,56 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
               isClearable: false,
               options: _.map(({ displayName, accountName }) => ({ label: displayName, value: accountName }), billingAccounts),
               onChange: ({ value: newAccountName }) => setSelectedBilling(newAccountName)
+            }),
+            div({ style: { marginTop: '1rem' } },
+              ['Note: Changing the billing account for this billing project will clear the workflow spend report configuration.'])
+          ])])
+        ])
+      ]),
+      div({ style: { color: colors.dark(), fontSize: 14, display: 'flex', alignItems: 'center', margin: '0.5rem 0 0 1rem' } }, [
+        span({ style: { flexShrink: 0, fontWeight: 600, fontSize: 14, marginRight: '0.75rem' } }, 'Workflow Spend Report Configuration:'),
+        span({ style: { flexShrink: 0 } }, 'Edit'),
```

Comment:
While it's technically acceptable to have a string as the child of an element, we prefer to pass it as an array for consistency. Also, inside a `display: 'flex'`, there's no reason to use a span instead of a div, since that's our usual go-to.